validate.ts

```diff
@@ -0,0 +1,52 @@
+// var validateform = index => {
+//     let error = false;
+
+//     if(!validate(index, "email") || !validate(index, "address") || !validate(index, "fname") || !validate(index, "lname") || !validate(index, "number"))
+//     error = true;
+
+//     return error;
+
+// }
+
+export const validate = (e: any): boolean => {
```

Comment:
no any please

Reply:
This is an InputEvent, but when this event is not recognized.